README.md

```diff
@@ -657,3 +657,30 @@ Chaining the [Amazon S3 Triggered Endpoint Imports](#Amazon-S3-Triggered-Endpoin
 ## License
 
 This library is licensed under the MIT-0 License. See the LICENSE file.
+
```

Comment:
Can you please move this under the: Amazon Pinpoint Basics section making sure to also include it in the Table of Contents above?  I tend to share the Table of Contents with customers so they have an idea of the types of things that are in this repo